.pylintrc

```diff
@@ -29,3 +29,8 @@ ignored-classes=numpy,scipy,autograd,toml,appdir,autograd,autograd.numpy,autogra
 # multiple time (only on the command line, not in the configuration file where
 # it should appear only once).
 disable=line-too-long,invalid-name,too-many-lines,redefined-builtin,too-many-locals,duplicate-code
+
+[MISCELLANEOUS]
+
+# List of note tags to take in consideration, separated by a comma.
+notes=
```

Comment:
Shouldn't we add TODO here if we wan't it to be ignored? I'm not entirely sure what "take in consideration" means in this context.

Reply:
No, by not listing anything, we're ignoring everything. :smile: If we added TODO here, then that would be considered. See here: https://docs.pylint.org/en/1.6.0/features.html#id9